website/pages/docs/reference/workspaces.mdx

```diff
@@ -0,0 +1,225 @@
+version = "1.2.3"
+authors = ["Nice Folks"]
+description = "A short description of my package"
+documentation = "https://example.com/bar"
+
+# [PROJECT_DIR]/bar/Scarb.toml
+[package]
+name = "bar"
+version.workspace = true
+authors.workspace = true
+description.workspace = true
+documentation.workspace = true
+```
+
+### `[dependencies]`
+
+The `workspace.dependencies` table is where you define dependencies to be inherited by members of a workspace.
+
+Specifying a workspace dependency is similar to package dependencies,
+except you can then inherit the workspace dependency as a package dependency
+
+Example:
+
+```
+# [PROJECT_DIR]/Scarb.toml
+[workspace]
+members = ["foo", "bar"]
+
+[workspace.dependencies]
+alexandria = { git = "https://github.com/keep-starknet-strange/alexandria.git" }
+openzeppelin = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", branch = "cairo-2" }
+
+# [PROJECT_DIR]/foo/Scarb.toml
+[package]
+name = "foo"
+version = "0.2.0"
+
+[dependencies]
+alexandria.workspace = true
+
+# [PROJECT_DIR]/bar/Scarb.toml
+[package]
+name = "bar"
+version = "0.2.0"
+
+[dependencies]
+openzeppelin.workspace = true
+```
+
+Note: paths used to declare path dependencies are relative to workspace root.
+
+### `[scripts]`
+
+The `[scripts]` section can be used to define custom, cross platform commands specific to the workspace codebase.
+The values from the `[workspace.scripts]` table are available to be inherited by members with `{key}.workspace = true`.
+
+Scripts are run for workspace member packages specified with `--package/--workspace` filters.
+
+For example:
+
+```
+# [PROJECT_DIR]/Scarb.toml
+[workspace]
+members = ["foo"]
+
+[workspace.scripts]
+test = "snforge"
+
+# [PROJECT_DIR]/foo/Scarb.toml
+[package]
+name = "foo"
+version = "0.2.0"
+
+[scripts]
+test.workspace = true
+```
+
+See [Scripts](./scripts) page for more information.
+
+### `[tool]`
+
+The `workspace.tool` table can be used for tools that would like to store configuration in Scarb.toml.
+Similarly to the `[tool]` section from the package manifest,
+the `[workspace.tool]` is not parsed by Scarb and will not be warned about.
+The values from the `[workspace.tool]` table are available to be inherited by members with `{key}.workspace = true`.
+
+For example:
+
+```
+# [PROJECT_DIR]/Scarb.toml
+[workspace]
+members = ["foo"]
+
+[workspace.tool.snforge]
+exit_first = true
+
+# [PROJECT_DIR]/foo/Scarb.toml
+[package]
+name = "foo"
+version = "0.2.0"
+
+[tool]
+snforge.workspace = true
+```
+
+See [Tool](./manifest#tool) page for more information.
+
+## `[profile]`
+
+In a workspace context, only profiles defined in the root manifest are applied.
+
+See [Profiles](./profiles) page for more information.
+
+## Acknowledgements
+
+This functionality is based on [Cargo workspaces](https://doc.rust-lang.org/cargo/reference/workspaces.html)
+and both the design and implementation has been heavily influenced by the approach suggested by the Cargo team.
```

Comment:
```suggestion
```

That's not documentation-style. I don't think we should acknowledge everything everywhere (I do actually also have mixed feelings about the way how we said it in deno_task_shell docs).